reV/batch/batch.py

```diff
@@ -64,18 +67,28 @@ def _parse_config(config):
             List of dictionaries representing the different arg/value
             combinations made available in the batch config json.
         file_sets : list
-            List of same length as arg_combs, representing the files to
-            manipulate for each arg comb.
+            List representing the files to manipulate for each arg comb
+            (same length as arg_combs).
         set_tags : list
-            List of strings of tags for each batch job set.
+            List of strings of tags for each batch job set
+            (same length as arg_combs).
         """
 
         arg_combs = []
         file_sets = []
         set_tags = []
+        sets = []
 
         # iterate through batch sets
         for s in config['sets']:
+            set_tag = s.get('set_tag', '')
+            if set_tag in sets:
+                msg = ('Found multiple sets with the same set_tag: "{}"'
+                       .format(set_tag))
+                logger.error(msg)
+                raise ValueError(msg)
+            else:
+                sets.append(set_tag)
```

Comment:
What happens if set_tag == ''

Reply:
That's the default set tag (no set tag). That's used to partition the permutations and to set up the job names. So the job names look like: `set_tag_{arg_combination_0}_{arg_combination_1}`. I also added a protection so that multiple sets cannot have the same set tag (and multiple sets cannot have undefined set tags): https://github.com/NREL/reV/pull/204/files#diff-6e2feea37f6607cee458554b892246a3R85